aiohttp/tracing.py

```diff
@@ -8,6 +9,15 @@
 from .signals import Signal
 
 
+if TYPE_CHECKING:
+    from .client import ClientSession
+    from .web_app import Application
+
+    _Signal = Signal[Callable[[Application], Awaitable[None]]]
```

Comment:
Please replace `Application` with `TraceConfig` to fix the signature

Reply:
done